assets/css/index.css

```diff
@@ -1,3 +1,7 @@
+html {
+  scroll-behavior: smooth;
```

Comment:
Never know this attribute, nice!

Reply:
I think this issue will be resolved with my pull request https://github.com/code-co-op/learning-web-development/pull/22. Also I believe we will be adding a background to the nav as we scroll down the past the header so you wouldn't be able to see the content behind the navbar.